.editorconfig

```diff
@@ -3,25 +3,30 @@ root = true
 [*]
 indent_style = space
 end_of_line = lf
-insert_final_newline = true
+insert_final_newline = false
 trim_trailing_whitespace = true
```

Comment:
Then I misunderstood this during a conversation with @jirikuncar. I'll remove that change.